redisinsight/api/src/modules/ai/chat/exceptions/conv-ai.error.handler.ts

```diff
@@ -0,0 +1,34 @@
+import { AxiosError } from 'axios';
+import { HttpException } from '@nestjs/common';
+import {
+  ConvAiBadRequestException,
+  ConvAiForbiddenException, ConvAiInternalServerErrorException,
+  ConvAiUnauthorizedException,
+} from 'src/modules/ai/chat/exceptions';
+import { ConvAiNotFoundException } from 'src/modules/ai/chat/exceptions/conv-ai.not-found.exception';
```

Comment:
I guess will be better to import ConvAiNotFoundException from 'src/modules/ai/chat/exceptions'; as well